### DISCUSSION TITLE
Searching By Metadata

### DISCUSSION THREAD

**Discussion by bmschwa** [formkiq/formkiq-core#118] [General]
Hi, 

If this is documented elsewhere, please point me in the right direction.

So, I started uploading some content to my instance.  I've set key/value tags in the past and learned that in FormKiq, tags are just values whereas metadata is Key/Value pairs.   

I think I know how to pull up all documents by a _tag_ ... Is there any way to search and get a list of documents by Metadata?

Thanks,

Brian

**Comment by reganwolfrom** [comment]
@bmschwa 
In the API, you should be able to access key/value tags (aka metadata) using key: for key and eq: for value (or eqOr for multiple values).

It looks like this IS NOT shown in our document search page, so we'll need to add it: https://docs.formkiq.com/docs/1.9.0/api/index.html#tag/Document-Search/operation/DocumentSearch


```{
  "query": {
    "tag": 
      {
        "eq": "string",
        "key": "string"
      }
    ]
  }
}
